packages/blockchain-wallet-v4-frontend/src/modals/Interest/InterestDetails/Summary/model.ts

```diff
@@ -0,0 +1,26 @@
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December'
+]
+
+export const getNextMonth = (): string => {
+  const current = new Date()
+  const next =
+    current.getMonth() === 11
+      ? new Date(current.getFullYear() + 1, 0, 1)
+      : new Date(current.getFullYear(), current.getMonth() + 1, 1)
+
+  return (
+    months[next.getMonth()] + ' ' + next.getDate() + ', ' + next.getFullYear()
```

Comment:
Can you use the `moment` library here instead? I think you could do just `moment.add(1, 'months')`.

https://momentjs.com/ (we already have the lib installed)

Reply:
Ahh forgot about that library. I will add changes.